src/watcher/watcher.ts

```diff
@@ -43,9 +43,14 @@ export function createWatcher(ctx: Context) {
   let includePaths: Array<RegExp> = [];
   let ignorePaths: Array<RegExp> = [];
 
+  // ensure root is string[]
+  const root = typeof props.root === "string" ? [props.root] : props.root
```

Comment:
[].concat(props.root)

Reply:
I avoided `[].concat(x)` because the type of that expression is `any[]`

![image](https://user-images.githubusercontent.com/158330/77642116-8397cb80-6f33-11ea-85ae-d04fe1a12e53.png)

![image](https://user-images.githubusercontent.com/158330/77641885-2734ac00-6f33-11ea-93ea-e828956a7ff5.png)

Reply:
Oh, also... more importantly `props.root` might be `undefined`